test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -505,11 +510,16 @@ def setUp(self):
             return
         self.log_enter_setup()
         self.init_proc_mon()
-        self.revert_servers()
-        self.revert_moms()
-        self.revert_comms()
-        self.revert_schedulers()
-        self.revert_pbsconf()
+        _, path = tempfile.mkstemp(prefix="setup_output",
+                                   suffix=".json")
+        self.saved_file = path
```

Comment:
won't this overwrite the 'saved_file' path that you created in setupclass() ?

Reply:
prefix of both the files are different so, i will not overwrite file created in setupclass